### DISCUSSION TITLE
ParaView software crashes when trying to open exodus files

### DISCUSSION THREAD

**Discussion by JFihosy** [idaholab/moose#21625] [Q&A General] [answered]
So I'm trying to use ParaView to view exodus files created by diffusion models I'm running in MOOSE, but the software crashes as soon as I try to open them.

Here's the input file I'm using:
[crash_input_example.txt](https://github.com/idaholab/moose/files/9140093/crash_input_example.txt)

Meanwhile ParaView is able to easily display output files from input files such as this:
[runs_fine_example.txt](https://github.com/idaholab/moose/files/9140103/runs_fine_example.txt)

If anyone has any ideas on why this might be happening it would be greatly appreciated. Thanks.

**Comment by jcstonehillAMA** [comment, marked as answer]
I had a very similar problem with the mesh generated by the FlowChannel1Phase components in the Thermal Hydraulics Module. In short, Paraview 5.10.1 didn't like 1D meshes from exodus files. 2D and 3D works fine, but 1D crashes Paraview 5.10.1. At the recommendation of a post somewhere on the internet I switched to Paraview 5.9.1 and everything worked as intended. I read that 5.10.1 has a legacy exodus reader that solves this problem, but rather than figuring out how to get that working I switched to 5.9.1.

Hopefully that's all that's wrong! Good luck!

EDIT: Apparently I read the solution here on this discussion board: https://github.com/idaholab/moose/discussions/21138

**Reply by JFihosy** [reply]
You were right! ParaView 5.9.1 works perfectly for me. Thank you.

I raised the issue with 5.10.1 with ParaView's support team. Apparently it's an issue with the IOSSReader. I don't know much more than that but hopefully this is patched in 5.10 soon.

**Comment by grmnptr** [comment]
You can try to use the legacy exodus reader in paraview:
`Tools > Manage Plugins > Legacy Exodus Reader > Select Plugin` > reopen your file and select the appropriate reader.
